modules/light-clients/07-tendermint/store.go

```diff
@@ -273,11 +273,11 @@ func GetPreviousConsensusState(clientStore sdk.KVStore, cdc codec.BinaryCodec, h
 
 // PruneAllExpiredConsensusStates iterates over all consensus states for a given
 // client store. If a consensus state is expired, it is deleted and its metadata
-// is deleted.
+// is deleted. The number of consensus states pruned is returned.
 func PruneAllExpiredConsensusStates(
```

Comment:
This function is only used for migrations atm so I thought it'd be useful to return the amount of consensus states pruned for logging purposes